bioptim/optimization/optimal_control_program.py

```diff
@@ -712,52 +712,69 @@ def load(file_path: str) -> list:
             out = [ocp, sol]
         return out
 
+    def get_dynamics(self):
+        list_dynamics = []
+        for nlp in self.nlp:
+            list_dynamics.append(nlp.dynamics_type.type.name)
+        return list_dynamics
+
+    def get_ode_solver(self):
+        list_ode = []
+        for nlp in self.nlp:
+            list_ode.append(nlp.ode_solver.rk_integrator.__name__)
+        return list_ode
+
     def structure_graph(
         self,
-        n_shooting: Union[int, tuple],
         print_to_terminal: bool = False,
```

Comment:
`print_to_terminal` should be True by default

Reply:
Done